src/index.html

```diff
@@ -72,6 +72,10 @@ <h3 class='sleep-latest-week-data sleep-data'></h3>
            <h2 class='sleep-achievements-title sleep-title'>Achievements</h2>
            <h3 class='sleep-achievements-data sleep-data'></h3>
          </div>
+         <div class='sleep-DIY-module sleep-module'>
```

Comment:
We can make this more accessible by changing the `div` to `section`!

Reply:
I need to do more research on what changes to make in terms of using `div`. Using more accessible layout elements is my new personal goal.